src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -374,11 +382,19 @@ public AgentSettings LoadSettings()
 #endif
                 }
 
-                //delete agent from the server
-                currentAction = StringUtil.Loc("UnregisteringAgent");
-                _term.WriteLine(currentAction);
                 bool isConfigured = _store.IsConfigured();
                 bool hasCredentials = _store.HasCredentials();
+
+#if OS_WINDOWS
```

Comment:
I would expect to see the unconfigure-auto-logon code follow the same pattern as the unconfigure-service code. Does this need to be different?